examples/pmatch.kl

```diff
@@ -5,45 +5,29 @@
 
 (import (shift "prelude.kl" 1))
 
-(import (shift "quasiquote.kl" 1))
+(import (shift "identifier.kl" 1))
+(import (shift "dot-dot-dot.kl" 1))
 (import (shift "defun.kl" 1))
 (import (shift "lispy-do.kl" 1))
 (import (shift "free-identifier-case.kl" 1))
 (import (shift "temporaries.kl" 1))
 (import (shift "pair-datatype.kl" 1))
 (import (shift "list-datatype.kl" 1))
 
-(meta
-  (define identifier?
-    (lambda (stx)
-      (syntax-case stx [(ident x) (true)] [_ (false)]))))
-
 (meta
   (datatype (Maybe A)
     (nothing)
     (just A)))
 
 (meta
   (defun syntax->list (stx)
-    (syntax-case stx
-      [() (nil)]
-      [(cons x xs) (:: x (syntax->list xs))])))
-
-(meta
-  (define else-case
```

Comment:
`else-case` is no longer needed, it's much easier to use a nested pattern on the `else` keyword.